sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DescribeNamespaceExec.scala

```diff
@@ -46,7 +46,8 @@ case class DescribeNamespaceExec(
     }
 
     if (isExtended) {
-      val properties = metadata.asScala -- CatalogV2Util.NAMESPACE_RESERVED_PROPERTIES
+      val properties = metadata.asScala.filterNot(
```

Comment:
how about `metadata.asScala.toMap --`, then the subsequent `properties` don't need to call `toMap`.

Reply:
Good suggestion!